missilewars-plugin/src/main/java/de/butzlabben/missilewars/commands/SetupCommands.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * This file is part of MissileWars (https://github.com/Butzlabben/missilewars).
+ * Copyright (c) 2018-2021 Daniel Nägele.
+ *
+ * MissileWars is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MissileWars is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MissileWars.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.butzlabben.missilewars.commands;
+
+import co.aikar.commands.BaseCommand;
+import co.aikar.commands.annotation.*;
+import de.butzlabben.missilewars.configuration.Config;
+import de.butzlabben.missilewars.configuration.Messages;
+import de.butzlabben.missilewars.game.Game;
+import de.butzlabben.missilewars.game.GameManager;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandAlias("mw|missilewars")
+@Subcommand("setup")
+public class SetupCommands extends BaseCommand {
+
+    private Game game;
+    private Player player;
+
+    @Default
+    @CommandPermission("mw.setup")
+    public void setupCommands(CommandSender sender, String[] args) {
+        sender.sendMessage(Messages.getPrefix() + "§fSetup usage: §7/mw setup <main|lobby|arena> <value> <set|teleport> [lobby]");
+    }
+
+    @Subcommand("main")
+    public class mainSetupCommands extends BaseCommand {
+
+        @Subcommand("fallbackspawn")
+        public class fallbackspawnSetup extends BaseCommand {
+
```

Comment:
With the setup commands some coordinates can be changed ingame. You can also teleport there.